pyproject.toml

```diff
@@ -100,3 +100,26 @@ line-ending = "auto"
 docstring-code-format = false
 
 docstring-code-line-length = "dynamic"
+
+[tool.poetry]
+name = "osipi"
+version = "0.1.2"
+description = "The authorative python package for perfusion MRI"
+authors = ["Luis Torres <luistorres@flywheel.io>", "Steven Sourbron <s.sourborn@sheffield.ac.uk>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.21.2"
+scipy = "^1.7.3"
+matplotlib = "^3.4.3"
+pre-commit = "^3.7.1"
+requests = "^2.32.3"
+
+[tool.poetry.dev-dependencies]
```

Comment:
We should include our linting deps here. rst-checker has a sphinx add-on that might help fix the docs issue in the pipeline.

Reply:
There are some errors related to sphinx after I added rst-checker with sphinx as extra, but it also gives some errors so I ignored them.
I think it's not a problem if we will write new docs using mkdocs?